app/javascript/react/screens/App/Plan/components/PlanRequestDetailList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { ListView, Spinner, Tooltip, UtilizationBar } from 'patternfly-react';
+import { IsoElpasedTime } from '../../../../../components/dates/IsoElapsedTime';
+
+const PlanRequestDetailList = ({ planRequestDetails }) => (
+  <ListView className="plan-request-details-list">
+    {planRequestDetails.tasks.map(task => {
+      let leftContent;
+      if (task.message === 'Pending') {
+        leftContent = <ListView.Icon name="pending" />;
+      } else if (task.message === 'VM Transformations completed') {
+        leftContent = <ListView.Icon type="fa" name="check" />;
+      } else {
+        leftContent = <Spinner loading />;
+      }
+      const currentTime = new Date();
+      const startDateTime = new Date(task.delivered_on);
+      const lastUpdateDateTime = new Date(task.updated_on);
+      const elapsedTime = IsoElpasedTime(
+        startDateTime,
+        task.completed ? lastUpdateDateTime : currentTime
+      );
+      const label = sprintf(
+        __('%s of %s Migrated'),
+        task.diskSpaceCompletedGb,
+        task.totalDiskSpaceGb
+      );
+
+      return (
+        <ListView.Item
+          key={task.id}
+          leftContent={leftContent}
+          heading={task.transformation_host_name}
+          additionalInfo={[
+            <div style={{ paddingRight: 60 }}>
+              <span>
+                <strong>Started: </strong>
```

Comment:
Translation support (here and well and other places)

Reply:
yup yup...added